package.json

```diff
@@ -34,8 +34,8 @@
 	"dependencies": {
 		"bluebird": "^3.7.2",
 		"chalk": "^1.1.3",
-		"ecmarkdown": "4.0.0",
-		"grammarkdown": "^2.1.2",
+		"ecmarkdown": "NEXT",
```

Comment:
why is the tag in all caps? it'll work but it's pretty unconventional

Reply:
Oh, the point is that I'm depending on an unpublished version, and I'll update this to the real version when it's published. That's not an actual tag.